data/uhc_pack/functions/pre_generation/stage/step/teleport/dynamic/do.mcfunction

```diff
@@ -0,0 +1,4 @@
+forceload add 0 0
+summon minecraft:armor_stand 0 250 0 {Invisible:1b,Invulnerable:1b,Small:1b,NoGravity:1b,Tags:["pre_generator","new"]}
+
+execute as @e[tag=pre_generator,tag=new,limit=1] run function uhc_pack:pre_generation/stage/step/teleport/dynamic/teleport
```

Comment:
For all the `@e[tag=pre_generator]` selectors I would prefer that we include `type=minecraft:armor_stand` as well as it should be a little more efficient and it is consistent with the rest of the `@e` selectors in the datapack